src/apps/concepts/components/FilterOptions.tsx

```diff
@@ -163,12 +164,14 @@ const FilterOptions: React.FC<FilterOptionsProps> = ({
 }) => {
   const classes = useStyles();
   const [searchText, setSearchText] = useState("");
+  const { replace: goTo } = useHistory();
 
   const clearAllFilters = () => {
     setCheckedClasses([]);
     setCheckedDataTypes([]);
     setCheckedGeneral([]);
     setCheckedSources([]);
+    goTo(url);
```

Comment:
I actually wonder about this... Right now the behaviour is that I click each filter and then click the "APPLY FILTERS" link to actually update the view. The logic behind the way the "CLEAR ALL" button currently work is that it's just clearing the filters as if you had cleared them manually. You still need to click "APPLY FILTERS" for the changes to make.

I think the question is: is the user experience expectation that clicking on the "CLEAR ALL" button removes all the filters *and* updates the table, or do we want to just clear all the filters and let the updating of the table take place after the user has reset their filters.

There are two reasons I might click on the "CLEAR ALL" button:

1. I really want to go back to the view of all concepts.
2. I just want to clear out the filters I previously had so I can create a new set of filters.

This new behaviour obviously makes the flow easy for 1, but is it the right flow for 2?

And in either case, I would update this PR to reflect what this change actually does (i.e. reloads the table when clicking "CLEAR ALL") rather than the now disconnected ticket it was first attached to.

Reply:
I think the "CLEAR Filter" button should remove all the filters, update the url and make a request since the URL has changed so that the user goes back to the view without filters but we should remember to keep `page and q` since they are not attached to the other filters.

Reply:
`q` makes sense to keep, but not `page` if we're making a new query (why would I want to be on page 5 of all the concepts after having been on page 5 of all the question concepts?)

Reply:
That would be great @ibacher if you can update this PR with that fix, I will update the ticket description to capture the clear filter changes.

Reply:
You are right @ibacher , now I see it for the `page` setting to `1`, though given where the clear filters button is placed it looks like it's only for clearing the filters in the sideBar and not anything else. But I think that's a minor UX thing, now that I think of it, it's probably what the user expects.

Reply:
Thanks, @hadijahkyampeire, and @ibacher I think we should leave it as it is for now. So, just closing the PR.